### PR TITLE
ref(ui): Various CSS cleanups in Issue Group Header

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/organizationGroupDetails/header.tsx [hunk 1]
```diff
@@ -56,12 +56,8 @@ type Props = WithRouterProps & {
   event?: Event;
 };
 
-type MemberList = NonNullable<
-  React.ComponentProps<typeof AssigneeSelector>['memberList']
->;
-
 type State = {
-  memberList?: MemberList;
+  memberList?: User[];
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:60]: 🤷

File: static/app/views/organizationGroupDetails/header.tsx [hunk 2]
```diff
@@ -344,36 +340,36 @@ class GroupHeader extends Component<Props, State> {
...
-          <BreadcrumbProjectBadge
+        <ShortIdBreadrcumb>
+          <ProjectBadge
             project={project}
             avatarSize={16}
             hideName
             avatarProps={{hasTooltip: true, tooltip: project.slug}}
           />
-          <StyledTooltip
+          <Tooltip
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:350]: Was unnecessary afaict

File: static/app/views/organizationGroupDetails/header.tsx [hunk 3]
```diff
@@ -344,36 +340,36 @@ class GroupHeader extends Component<Props, State> {
 
     const shortIdBreadCrumb = group.shortId && (
       <GuideAnchor target="issue_number" position="bottom">
-        <IssueBreadcrumbWrapper>
-          <BreadcrumbProjectBadge
+        <ShortIdBreadrcumb>
+          <ProjectBadge
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:344]: Also not needed if we use flex w/ gap on the `ShortIdBreadrcumb`

File: static/app/views/organizationGroupDetails/header.tsx [hunk 4]
```diff
@@ -384,69 +380,57 @@ class GroupHeader extends Component<Props, State> {
...
               <TitleWrapper>
                 <h3>
                   <StyledEventOrGroupTitle hasGuideAnchor data={group} />
                 </h3>
-                {group.inbox && (
-                  <InboxReasonWrapper>
-                    <InboxReason inbox={group.inbox} fontSize="md" />
-                  </InboxReasonWrapper>
-                )}
+                {group.inbox && <InboxReason inbox={group.inbox} fontSize="md" />}
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:389]: Another thing that can be solved w/ gap

File: static/app/views/organizationGroupDetails/header.tsx [hunk 5]
```diff
@@ -384,69 +380,57 @@ class GroupHeader extends Component<Props, State> {
...
+                        </Link>
+                      </EventAnnotation>
+                    )
                   }
                 />
               </StyledTagAndMessageWrapper>
             </div>
-
             <StatsWrapper>
-              <div className="count align-right m-l-1">
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:425]: fix w/ gap

File: static/app/views/organizationGroupDetails/header.tsx [hunk 6]
```diff
@@ -384,69 +380,57 @@ class GroupHeader extends Component<Props, State> {
...
+              <div className="count">
                 <h6 className="nav-header">{t('Events')}</h6>
-                {disableActions ? (
+                <Link disabled={disableActions} to={eventRouteToObject}>
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:416]: can just set `disabled` on the link

File: static/app/views/organizationGroupDetails/header.tsx [hunk 7]
```diff
@@ -456,18 +440,20 @@ class GroupHeader extends Component<Props, State> {
                 />
               </div>
             </StatsWrapper>
-          </div>
-          <SeenByList
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/header.tsx:460]: gross `float` where the element comes first 🤮

File: static/app/components/seenByList.tsx [hunk 8]
```diff
@@ -45,10 +44,7 @@ const SeenByList = ({
 
   // Note className="seen-by" is required for responsive design
```
Comment (malwilley) [inline static/app/components/seenByList.tsx:45]: Don't think this comment is needed anymore, right?

**Review by k-fish — APPROVED** [review]

**Review by malwilley — COMMENTED** [review]
Looks like an acceptance test is failing

Also, some regressions - actions disappear and events/users/assignee should be right-aligned: 

image

**Comment by evanpurkhiser** [comment]
> events/users/assignee should be right-aligned

I specifically changed this bc it looks gross on the right in this layout

image

**Pull request merged**